week-2/L-array-methods-2/exercise2.js

```diff
@@ -8,12 +8,13 @@
 */
 
 function capitalise(str) {}
+var name = ["D", "a", "n", "i", "e", "l"];
+var arr = name.join();
```

Comment:
remember, you have to pass WHAT you want it to be joined on. instead of `.join()`, try `.join('')`